### PR TITLE
added Cataract detection

### PR DESCRIPTION
**Pull request by ThilakShekharShriyan**
Changes were made to add the cataract-detection model File.

Have added the data set used as well please review and let us know if it is needed , The data set includes the pre processed images that we use to train

### CODE REVIEW

**Comment by iArpanPatel** [comment]
Hi @ThilakShekharShriyan, Cataract detection example has been added as submodule, and there is no files inside. Please correct this.

**Comment by ThilakShekharShriyan** [comment]
Hi @iArpanPatel  Apologies for the delayed response. Checking

**Comment by ThilakShekharShriyan** [comment]
Hi @iArpanPatel , Can you please check if the required files are added and sufficient.

File: examples/cataract-detection/cataract.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+default_min_peers = 2
+
+
+def main():
+    dataDir = os.getenv('DATA_DIR', '/platform/swarmml/data')
+    modelDir = os.getenv('MODEL_DIR', '/platform/swarmml/model')
+    max_epochs = int(os.getenv('MAX_EPOCHS', str(default_max_epochs)))
+    min_peers = int(os.getenv('MIN_PEERS', str(default_min_peers)))
+
+    (x_train, y_train), (x_test, y_test) = load_data(dataDir)
```
Comment (iArpanPatel) [inline examples/cataract-detection/cataract.py:20]: Definition for load_data is missing.

File: examples/cataract-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+docker build -t user-ml-env-tf2.7.0 --build-arg https_proxy=http://<your-proxy-server-ip>:<port> workspace/cataract-detection/ml-context
+```
+
+6.  On host-1, Run Swarm Network node \(sentinel node\)
+
+```
+./scripts/bin/run-sn -d --name=sn1 --host-ip=172.1.1.4 \
+-e http_proxy= -e https_proxy=  \
+--sentinel --sn-api-port=30304 --key=demo/cataract-detection/cert/sn-1-key.pem \
+--cert=demo/cataract-detection/cert/sn-1-cert.pem \
```
Comment (iArpanPatel) [inline examples/cataract-detection/README.md:109]: cert path should be workspace/cataract-detection/cert/...
This needs to be corrected in all the run commands.

**Comment by ThilakShekharShriyan** [comment]
Hi @iArpanPatel , I see few comments in https://github.com/HewlettPackard/swarm-learning/pull/252 Do i need to make any changes.

**Comment by iArpanPatel** [comment]
Hi @ThilakShekharShriyan, you dont need to make any changes in #252. As your PR (#165) is based on old directory structure, we have taken your updates and raised a new PR #252 with some corrections. Thanks for your contribution, once #252 is merged we will close this (#165) PR and give you credit for the contribution.

**Comment by iArpanPatel** [comment]
Thanks for responding to the queries and for your contribution to HPE Swarm Learning. Due to changes in the examples directory structure, your submission has been merged under https://github.com/HewlettPackard/swarm-learning/pull/252 with some corrections. 
Closing this PR.

**Pull request closed without merging**